CONTRIBUTING.rst

```diff
@@ -58,6 +58,8 @@ Coding Guidelines
 
 - Comments and names of variables/functions/etc. must be in English
 
+- Formatting scripts are available `here <./build-aux>`__
```

Comment:
Should probably include the detail that those scripts are mainly meant for macOS and Linux for the time being. Otherwise a Windows-based developer might follow this advise and come up empty.